### PR TITLE
Adding bg-mpl-stylesheets

### PR DESCRIPTION
**Pull request by connorjbracy**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by connorjbracy** [comment]
I @connorjbracy am willing to be a maintainer for this package.

@sbillinge please help satisfy PR checklist item: 

- [ ] "GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there."

for both `sbillinge` and `Billingegroup`

**Comment by sbillinge** [comment]
I @sbillinge am willing to be a maintainer for this package.

**Comment by sbillinge** [comment]
I @billingegroup am willing to be a maintainer for this package

File: recipes/bg-mpl-stylesheets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% set name = "bg-mpl-stylesheets" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c501bf584feb46f792c3dbf515ebb2ae97da2cfdf07baad1cbd952baf4a52720" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (xhochy) [inline recipes/bg-mpl-stylesheets/meta.yaml:11]: fn is not needed anymore
```suggestion
```

File: recipes/bg-mpl-stylesheets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set group = "Billingegroup" %}
+{% set name = "bg-mpl-stylesheets" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c501bf584feb46f792c3dbf515ebb2ae97da2cfdf07baad1cbd952baf4a52720" %}
```
Comment (xhochy) [inline recipes/bg-mpl-stylesheets/meta.yaml:4]: Don't use that as a variable anymore
```suggestion
```

File: recipes/bg-mpl-stylesheets/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/{{ group }}/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/bg-mpl-stylesheets/meta.yaml:13]:
```suggestion
  sha256: c501bf584feb46f792c3dbf515ebb2ae97da2cfdf07baad1cbd952baf4a52720
```

File: recipes/bg-mpl-stylesheets/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  home: https://github.com/{{ group }}/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Defines a standardized matplotlib stylesheet for the Billinge Group'
+
+extra:
+  recipe-maintainers:
+    - sbillinge
+    - Billingegroup
```
Comment (xhochy) [inline recipes/bg-mpl-stylesheets/meta.yaml:42]: This is an org, this thus probably cannot be a maintainer.

Comment (connorjbracy) [reply recipes/bg-mpl-stylesheets/meta.yaml:42]: Okay
```suggestion
```

**Pull request merged**